### PR TITLE
Component-Edidiong:Onboarding-Quote-Card-Component

### PR DESCRIPTION
**Pull request by edidee**
talentYardRec2

### CODE REVIEW

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please add a recording of your component. Let's see the output.

**Review by Remi-dee — APPROVED** [review]

**Comment by edidee** [comment]
@annonymousauthority  There’s a gif showing the output. I added it when I created the PR

File: Developers/app/components/onboardTechies/onboardingQuoteCard.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+
+function OnboardingQuoteCard() {
+  return (
+    <div className="max-w-[22rem] md:max-w-md mx-auto px-4 pt-4 pb-8 bg-dark rounded-3xl">
```
Comment (annonymousauthority) [inline Developers/app/components/onboardTechies/onboardingQuoteCard.js:6]: Could you remove the widths from the class as I said earlier on? Don't worry about it the responsiveness.

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Few changes

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**